databook/background/Mouse-visual-system.md

```diff
@@ -45,6 +45,26 @@ visual areas each contain a distinct map of {term}`Retinotopy`, i.e. each visual
 area has its own map of visual space. The borders between higher visual areas
 can be identified based on reversals in the retinotopic map at area boundaries.
 
+The retinotopic map is often studied using intrinsic signal imaging (ISI), which
```

Comment:
We use intrinsic signal imaging (ISI to create a retinotopic map, utilizing the difference ...